frame/vesting/src/lib.rs

```diff
@@ -168,39 +267,49 @@ pub mod pallet {
 				let locked = balance.saturating_sub(liquid);
 				let length_as_balance = T::BlockNumberToBalance::convert(length);
 				let per_block = locked / length_as_balance.max(sp_runtime::traits::One::one());
```

Comment:
Integrity test for `MaxVestingSchedules` > 0

Reply:
Also comparing minvestedtransfer w/ ed

Reply:
f1a760e